src/renderer/components/AddSliderDivider.tsx

```diff
@@ -8,11 +8,13 @@ import '../styles/AddSliderDivider.scss';
 interface IAddSliderDividerProps {
   sliderIndex: number;
   isMaxSliderCount: boolean;
+  style?: object;
```

Comment:
Let's change the type for this from `object` to `CSSProperties` (an import from `react`).